test/functional/commands/find/by-uiautomator-e2e-specs.js

```diff
@@ -11,111 +12,119 @@ describe('Find - uiautomator', function () {
   let driver;
   before(async () => {
     driver = await initDriver(APIDEMOS_CAPS);
-    await driver.implicitWait(20000);
+    await B.delay(20000);
   });
   after(async () => {
-    await driver.deleteSession();
+    await driver.quit();
   });
   it('should find elements with a boolean argument', async () => {
-    await driver.findElOrEls('-android uiautomator', 'new UiSelector().clickable(true)', true)
+    await driver.elementsByAndroidUIAutomator('new UiSelector().clickable(true)')
       .should.eventually.have.length.at.least(10);
   });
   it('should find elements within the context of another element', async () => {
     let els = await driver
-      .findElOrEls('-android uiautomator', 'new UiSelector().className("android.widget.TextView")', true);
+      .elementsByAndroidUIAutomator('new UiSelector().className("android.widget.TextView")');
     els.length.should.be.above(8);
     els.length.should.be.below(14);
   });
   it('should find elements without prepending "new UiSelector()"', async () => {
-    await driver.findElOrEls('-android uiautomator', '.clickable(true)', true)
+    await driver.elementsByAndroidUIAutomator('.clickable(true)')
       .should.eventually.have.length.at.least(10);
   });
   it('should find elements without prepending "new UiSelector()"', async () => {
```

Comment:
This test is completely duplicate of the above one. I think we can safely remove it.